mne/io/ctf/ctf.py

```diff
@@ -129,6 +129,7 @@ def __init__(self, directory, system_clock='truncate', preload=False,
             if sample_info['n_samp'] == 0:
                 break
             if len(fnames) == 0:
+                # XXX: Buffer size different to c version
                 info['buffer_size_sec'] = \
                     sample_info['block_size'] / info['sfreq']
```

Comment:
This is different from the c conversion tool. In effect it means that ``buffer_size_sec`` is different when reading with python version if ``n_trial`` == 1. I don't know the consequences of changing this so I didn't do it. Changing it also makes our tests fail.